cmd/ocsp-updater/main.go

```diff
@@ -508,9 +509,12 @@ func (l *looper) tick() {
 	sleepDur := expectedTickEnd.Sub(tickEnd)
 	if err != nil {
 		l.stats.Inc(fmt.Sprintf("OCSP.%s.FailedTicks", l.name), 1, 1.0)
-		if _, ok := err.(core.ServiceUnavailableError); ok && (l.failureBackoffFactor > 0 && l.failureBackoffMax > 0) {
-			l.failures++
-			sleepDur = core.RetryBackoff(l.failures, l.tickDur, l.failureBackoffMax, l.failureBackoffFactor)
+		if cfErr, ok := err.(*cferr.Error); ok {
```

Comment:
I'm pretty sure the cferr.Error type doesn't make it through the RPC layer.

I think probably what we want here it to back off regardless of error type. In the past we were worried that we would have a single response that would refuse to sign for some reason, and we would fail to make all progress. But after some operational experience I think that is unlikely to happen, and it's better than trying to enumerate all the error codes that are worth backing off.